.github/workflows/build_test.yml

```diff
@@ -259,24 +264,23 @@ jobs:
           install: >-
             base-devel
             git
-            mingw-w64-${{ matrix.arch }}-brotli
-            mingw-w64-${{ matrix.arch }}-cmake
-            mingw-w64-${{ matrix.arch }}-gcc
-            mingw-w64-${{ matrix.arch }}-gflags
-            mingw-w64-${{ matrix.arch }}-giflib
-            mingw-w64-${{ matrix.arch }}-gtest
-            mingw-w64-${{ matrix.arch }}-libavif
-            mingw-w64-${{ matrix.arch }}-libjpeg-turbo
-            mingw-w64-${{ matrix.arch }}-libpng
-            mingw-w64-${{ matrix.arch }}-libwebp
-            mingw-w64-${{ matrix.arch }}-ninja
+          pacboy: >-
+            brotli:p
+            cmake:p
+            gflags:p
+            giflib:p
+            gtest:p
+            libavif:p
+            libjpeg-turbo:p
+            libpng:p
+            libwebp:p
+            ninja:p
+            toolchain:p
 
```

Comment:
Nice, a much more intuitive action now 👍

Just out of curiousity, why was the entire `toolchain:p` needed? Usually just `cc:p` and (for C++ code) `gcc-libs:p` suffice, and might give some time savings by not installing all the toolchain packages... (Note that there is a `gcc-libs` metapackage available in clang environments for this reason, which will pull in libc++ and libunwind for you; there might missing corner cases though like openmp and libssp that are not part of that metapackage, do that's why I ask...)

Reply:
`clang64` environment and with `cc` was missing libs. Will try with `gcc-libs` (well, package name is counterintuitive)

Reply:
Indeed, hence the lengthy explanation 😉 It was one way to extend the legacy mingw environments...

Reply:
What's suspicious is that it tries to link 32-bit libs...

Reply:
The "32" is just the Win32 API, not the architecture...

Try adding `crt:p` to the list as well?